### PR TITLE
DOCS-2413 Synthetic Settings Screenshots Update

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Update two screenshots (Specify Value, Create from HTTP Tests) that include XPath functionality in Synthetics Settings.

### Motivation

DOCS-2413

### Preview

https://docs-staging.datadoghq.com/alai97/xpath-support-screenshot-update/synthetics/settings

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/settings/_index.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Choose the type of variable you want to create:
 4. Enter the **Value** you want to assign to your variable.
 5. Enable obfuscation of your variable to hide its value on test results (optional).
 
-{{< img src="synthetics/settings/variable_value.png" alt="Global Variable Specify Value"  style="width:100%;">}}
+{{< img src="synthetics/settings/variable_value_2.png" alt="Global Variable Specify Value"  style="width:100%;">}}
```
Comment (ruthnaebeck) [inline content/en/synthetics/settings/_index.md:45]:
```suggestion
{{< img src="synthetics/settings/variable_value_2.png" alt="Global Variable Specify Value" style="width:100%;">}}
```

File: content/en/synthetics/settings/_index.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ You can create variables from your existing [HTTP tests][1] by parsing their ass
     * Extract the value from **Response Header**: use the full response header for your variable or parse it with a [regex][2].
     * Extract the value from **Response Body**: parse the response body of the request with a [regex][2], a [`jsonpath`][3], an [`xpath`][4], or use the full response body.
 
-{{< img src="synthetics/settings/variable_fromhttp.png" alt="Variable from http"  style="width:100%;">}}
+{{< img src="synthetics/settings/variable_fromhttp_2.png" alt="Variable from http"  style="width:100%;">}}
```
Comment (ruthnaebeck) [inline content/en/synthetics/settings/_index.md:62]:
```suggestion
{{< img src="synthetics/settings/variable_fromhttp_2.png" alt="Variable from http" style="width:100%;">}}
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]

**Review by ruthnaebeck — APPROVED** [review]

**Comment by MargotLepizzera** [comment]
@alai97 Thanks for putting that PR together! Some comments:

- `Settings` page: do you think you could add an example of a variable actually being created from a JSON payload? Similar to what currently exists (with an actual JSONPath and preview). If you need help getting one working, don't hesitate to schedule some time on my cal for me to show you! 
- The XPath link is linking to RBAC instead of to some XPath doc.
- We would also need to update the [Multistep API test doc](https://docs.datadoghq.com/synthetics/multistep?tab=requestoptions#extract-variables-from-the-response) as well as on the [HTTP step](https://docs.datadoghq.com/synthetics/browser_tests/actions#extract-a-variable-from-the-response) for Browser tests.

**Comment by alai97** [comment]
@MargotLepizzera are you asking if we can add a video instead of a screenshot in the **Create from HTTP** section? If that's the case, can I ask you to provide me that content? 🙏🏼

The Multistep API Test screenshot is updated in PR #11307 (https://github.com/DataDog/documentation/pull/11307/files)! 

I'm updating the XPath link and adding a video for Browser Tests in the following commit.

**Comment by MargotLepizzera** [comment]
Ok so here are some links you can use @alai97 :

Settings - Create from HTTP test: https://a.cl.ly/L1urv2nG

Browser tests - HTTP step - Setup https://a.cl.ly/9ZuQLBRP
Browser tests - HTTP step - Add assertions https://a.cl.ly/NQuY517A
Browser tests - HTTP step - Extract variable https://a.cl.ly/P8uGDy6E

Multistep API tests - Define requests - Define the request https://a.cl.ly/Blux21jL
Multistep API tests - Define requests - Add assertion https://a.cl.ly/bLuqywY4
Multistep API tests - Define requests - Extract variables from response https://a.cl.ly/jku46Dr6

File: content/en/synthetics/browser_tests/actions.md [hunk 3]
```diff
@@ -335,10 +335,10 @@ To parse your variable:
...
 2. Decide whether to extract your variable from the response headers, or from the response body:
 
-    * Extract the value from **response header**: use the full response header of your HTTP request as the variable value or parse it with a [regex][15].
-    * Extract the value from **response body**: use the full response body of your HTTP request as the variable value, parse it with a [regex][15], a [`JSONPath`][13], or a [`XPath`][14].
+    * Extract the value from **response header**: use the full response header of your HTTP request as the variable value or parse it with a [`regex`][15].
+    * Extract the value from **response body**: use the full response body of your HTTP request as the variable value, parse it with a [`regex`][15], a [`JSONPath`][13], or a [`XPath`][14].
 
-{{< img src="synthetics/browser_tests/browser_test_vft.mp4" alt="Create a variable from HTTP request in Browser test" video="true"  width="80%" >}}
+{{< img src="synthetics/browser_tests/browser_test_vft2.mp4" alt="Create a variable from an HTTP request in a Browser test" video="true"  width="80%" >}}
```
Comment (alai97) [inline content/en/synthetics/browser_tests/actions.md:341]:
```suggestion
{{< img src="synthetics/browser_tests/browser_test_vft2.mp4" alt="Create a variable from an HTTP request in a Browser test" video="true" width="80%" >}}
```

File: content/en/synthetics/browser_tests/actions.md [hunk 4]
```diff
@@ -320,27 +322,29 @@ Optionally, you can base your step success on assertions about the defined HTTP
...
-    * Extract the value from **response header**: use the full response header of your HTTP request as the variable value or parse it with a [regex][15].
-    * Extract the value from **response body**: use the full response body of your HTTP request as the variable value, parse it with a [regex][15], a [`JSONPath`][13], or a [`XPath`][14].
+    * Extract the value from **response header**: use the full response header of your HTTP request as the variable value or parse it with a [`regex`][15].
+    * Extract the value from **response body**: use the full response body of your HTTP request as the variable value, parse it with a [`regex`][15], a [`JSONPath`][13], or a [`XPath`][14].
 
-{{< img src="synthetics/browser_tests/browser_test_vft.mp4" alt="Create a variable from HTTP request in Browser test" video="true"  width="80%" >}}
+{{< img src="synthetics/browser_tests/browser_test_vft2.mp4" alt="Create a variable from an HTTP request in a Browser test" video="true" width="80%" >}}
 
-Once created this variable can be used in the following steps of your browser test.
+Once created, this variable can be used in the following steps of your browser test.
```
Comment (MargotLepizzera) [inline content/en/synthetics/browser_tests/actions.md:347]: Add link to following section?

File: content/en/synthetics/multistep.md [hunk 5]
```diff
@@ -118,16 +117,16 @@ Assertions define what an expected test result is. When hitting `Test URL` basic
...
+| body          | `contains`, `does not contain`, `is`, `is not`, <br> `matches`, `does not match`, <br> [`jsonpath`][10], [`xpath`][11] | _String_ <br> _[Regex][12]_ <br> _String_, _[Regex][12]_ |
+| header        | `contains`, `does not contain`, `is`, `is not`, <br> `matches`, `does not match`                       | _String_ <br> _[Regex][12]_                                      |
 | response time | `is less than`                                                                                         | _Integer (ms)_                                                  |
 | status code   | `is`, `is not`                                                                                         | _Integer_                                                      |
 
 **Note**: HTTP tests can decompress bodies with the following `content-encoding` headers: `br`, `deflate`, `gzip`, and `identity`.
 
 You can create up to 20 assertions per step by clicking on **New Assertion** or by clicking directly on the response preview:
```
Comment (MargotLepizzera) [inline content/en/synthetics/multistep.md:127]:
```suggestion
You can create up to 20 assertions per step by clicking on **New Assertion** or by clicking directly on the response preview.
```

File: content/en/synthetics/multistep.md [hunk 6]
```diff
@@ -62,7 +62,7 @@ Hit **Create Your First Request** to start designing your test's requests.
 
 #### Define the request
 
-{{< img src="synthetics/api_tests/ms_define_request.png" alt="Define request for your Multistep API test" style="width:90%;" >}}
+{{< img src="synthetics/api_tests/ms_define_request2.png" alt="Define request for your Multistep API test" style="width:90%;" >}}
```
Comment (MargotLepizzera) [inline content/en/synthetics/multistep.md:65]: Let's move the image at the end of the section!

**Comment by MargotLepizzera** [comment]
Looking good @alai97! Added a few comments inline and just one last note about the updates on the browser test page: it looks like the [main HTTP request section](https://docs-staging.datadoghq.com/alai97/xpath-support-screenshot-update/synthetics/browser_tests/actions#http-requests) and the [setup section ](https://docs-staging.datadoghq.com/alai97/xpath-support-screenshot-update/synthetics/browser_tests/actions#set-up) are still using old screenshots? The [extract variable from response section](https://docs-staging.datadoghq.com/alai97/xpath-support-screenshot-update/synthetics/browser_tests/actions#extract-a-variable-from-the-response) also seems to have one screenshot and one recording - why not using just the new screenshot?

File: content/en/synthetics/browser_tests/actions.md (hunk 4 above)
Comment (alai97) [reply content/en/synthetics/browser_tests/actions.md:347]:
```suggestion
Once created, this variable can be used in the [following steps][#using-variables] of your browser test.
```

File: content/en/synthetics/multistep.md [hunk 7]
```diff
@@ -270,11 +269,11 @@ A test is considered `FAILED` if a step does not satisfy one or several assertio
...
-[13]: /monitors/notifications/?tab=is_alert#notification
-[14]: http://daringfireball.net/projects/markdown/syntax
-[15]: /monitors/notifications/?tab=is_recoveryis_alert_recovery#conditional-variables
-[16]: /synthetics/settings/#global-variables
-[17]: /synthetics/api_tests/errors/#ssl-errors
-[18]: https://www.w3schools.com/xml/xpath_syntax.asp
\ No newline at end of file
+[11]: https://www.w3schools.com/xml/xpath_syntax.asp
+[12]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions
+[13]: /api/v1/synthetics/#create-a-test
```
Comment (alai97) [inline content/en/synthetics/multistep.md:274]:
```suggestion
[13]: /api/latest/synthetics/#edit-an-api-test
```

**Pull request merged**